src/theme/theme.js

```diff
@@ -27,7 +27,7 @@ const theme = {
     body: 'Roboto, sans-serif',
   },
   space: [0, 5, 10, 15, 20],
-  breakpoints: ['375px', '768px', '1024px', '1440px'],
+  breakpoints: ['375px', '768px', '1280px', '1440px'],
```

Comment:
Why was this changed? I think it makes more sense at 1024, and the web site looks better for screen sizes between 1024 and 1280 when it's at 1024.

Reply:
the design file specifies these sizes. Also, it breaks the grid as there is a min-width property on the container for this bp at 1100px and 1024 is less than that so it ends up cut-off the screen.

Reply:
The grid is already flexbox based so shouldn't be an issue. I think we can prob set it to 1024 but also modify the the min-width for the container to be 960 so it doesn't break.